markdown/3.x/en/api/ff-asn.api.md

```diff
@@ -25,6 +25,7 @@ ___
 | **select-box**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | Use this when the hiddenLinks should be an HTML select element |
 | **disable-auto-expand**&nbsp;(Boolean) | Prevents group from being automatically expanded when it contains any active filters. |
 | **not-searchable**&nbsp;(Boolean) (default: false) | Prevents group from being searchable even if it satisfies `searchable-from` condition.  |
+| **topic**&nbsp;(String) | Defines if the component should subscribe to a custom topic instead of the default one. Please note that in this case you need to manually dispatch asn data.  For more information regarding the data's structure, please refer to **Dispatching data manually** in the [ff-asn documentation](/api/3.x/ff-asn#tab=docs). |
```

Comment:
`topic` should be under `ff-asn`. It currently appears to be under `ff-asn-group`.
I also noted that there is no mention of the `subscribe` attribute. Would you add that one, too, if possible?

Reply:
Fixed.
I added `subscribe` attribute, but also noticed that its type is `string`. in all occurrences. Should I change it to Boolean?

Reply:
Didn't check all occurrences but from what I found StringyBooleans are noted as type `String`.
`Boolean` is reserved to true boolean attributes as far as I know.